specification/compute/resource-manager/readme.md

```diff
@@ -271,6 +271,23 @@ suppressions:
     from: diagnostic.json
 ```
 
+### Tag: package-2024-06-01-preview
+
+These settings apply only when `--tag=package-2024-06-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-06-01-preview'
+input-file:
+  - Microsoft.Compute/DiagnosticRP/preview/2024-06-01-preview/diagnostic.json
+```
+
+### Tag: package-2024-06-01-preview-only
```

Comment:
> package-2024-06-01-preview-onl [](http://example.com/codeflow?start=8&length=31)

this seams a duplication of the package-2024-06-01-preview. is this intentional?

Reply:
Removed the tag, since there are no additional APIs, and the tags are redundant.
The preview-only tag was used for previous versions by Azure codegen tools to build packages for only APIs for that specific version.